x-pack/plugins/index_management/__jest__/components/index_table.test.js

```diff
@@ -254,15 +264,17 @@ describe('index table', () => {
     expect(button.text()).toEqual('Manage 2 indices');
   });
 
-  test('should show system indices only when the switch is turned on', async () => {
+  test('should show hidden indices only when the switch is turned on', async () => {
     const rendered = mountWithIntl(component);
     await runAllPromises();
     rendered.update();
 
-    snapshot(rendered.find('.euiPagination li').map((item) => item.text()));
-    const switchControl = rendered.find('.euiSwitch__button');
+    snapshot(namesText(rendered));
```

Comment:
Not sure why we were snapshotting the eui pagination list. I changed it to snapshot the list of indices names.